app/services/scorecard_endpoint_service.js

```diff
@@ -17,12 +17,12 @@ const scorecardEndpointService = (() => {
   }
 
   async function handleUpdateScorecardWithoutEndpointUrl() {
-    if (!reLoginService.isAppVersionForUpdateScorecard())
+    if (!reLoginService.isRequireReLogin())
```

Comment:
@limkimsan please consider introducing isReloggedIn() method and replacing it as the update empty endpoint URL didn't relevant to the isRequiredReloggedIn but it is relevant to the app is re-logged in instead.

if(!reLoginService.isReloggedIn()) {
    return;
}